tools/publish_aws.py

```diff
@@ -56,9 +56,11 @@ def __init__(
         logger.info('Using artifact bucket: {}'.format(s3_bucket))
 
         s3_dir_path = os.environ.get('S3_DIR_PATH', 'autodelete7d')
-        dir_name = '{}-{}'.format(
-            time.strftime("%Y%m%d-%H%M%S"),
-            ''.join([random.SystemRandom().choice(string.ascii_letters + string.digits) for i in range(16)]))
+        s3_dir_name = os.environ.get('S3_DIR_NAME')
```

Comment:
Not that it's critical, but we could break out the handling of these three variables to a function that returns a tuple `(bucket, dir_path, dir_name)`? Or even *just* `(s3_directory_url, http_directory_url)`.

Reply:
I've given it a go... looks OK to me, let me know what you think.